### PR TITLE
Fixed space under table icons on hover.

### PR DESCRIPTION
**Pull request by HeliumSquid**
When you hover over the table icons, there is a small space at the bottom.

http://bulma.io/documentation/components/tables/

Use to look like this:
hover

Now looks like this:
capture

### CODE REVIEW

**Comment by jgthms** [comment]
Thanks but the problem is that there's a space on normal links now:
screen shot 2016-02-27 at 1 32 04 pm

Still trying to figure out why it happens.

**Comment by HeliumSquid** [comment]
Sorry about that. I think I've found a solution now.
It should look like this now:

capture

**Comment by andrkrn** [comment]
I think this is because the `fa` class has height fixed at `24px` so there 1px space at the bottom.
When you change the height to `25px` or `100%` it should be fine.

**Comment by HeliumSquid** [comment]
Thanks @andrkrn that solved the issue.

File: bulma/components/table.sass [hunk 1]
```diff
@@ -16,7 +16,7 @@
       padding: 0
       & > a
         display: block
-        padding: 8px 10px
+        padding: 8.8px 10px
```
Comment (jgthms) [inline bulma/components/table.sass]: This feels like a huge hack.

Comment (jonnyreeves) [inline bulma/components/table.sass]: CSS you mean? ;)

Comment (seripap) [inline bulma/components/table.sass]: +1

**Comment by HeliumSquid** [comment]
Oops, sorry about that. I've now set the `.table td.table-icon .fa`'s height to `25px`.

**Pull request closed without merging**